Frontend/pointmapVuejs/.babelrc

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
General comment about the whole frontend - it should not live in a subfolder like this - the Frontend directory should be the base of the frontend project.

Reply:
@ChrisMeyer7088